collections.go

```diff
@@ -26,7 +26,7 @@ func (repository collectionRepository) Get(id int64) (shopify.Collection, error)
 
 	body, _, err := repository.client.Get(url, nil)
 	if err != nil {
-		return shopify.Collection{}, err
+		return shopify.Collection(nil), err
```

Comment:
return shopify.CustomCollection{} here

Reply:
yeah, I tried this, however I was getting a `invalid composite literal type error` when using shopify.collection{}. apparently using this `shopify.Collection(nil)` fixes it

Reply:
hmmm but the pipelines seem to fail, the vetter doesnt want us to do it